app-crypt/openpgp-keys-worker/openpgp-keys-worker-20210320.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Ralf Hoffmann's openPGP key used to sign app-misc/worker releases"
+HOMEPAGE="http://www.boomerangsworld.de/cms/about.html"
+SRC_URI="
+	https://keys.openpgp.org/vks/v1/by-fingerprint/F9299EE90A729029E71AF26B667132D0FBC52B37
+		-> RalfHoffmann.asc
+"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~hppa ppc ~ppc64 x86"
```

Comment:
I need to check what we do for KEYWORDS for verify-sig but I suspect we need to just copy the keywords of app-misc/worker, "ekeyword ~all", then stabilise with it in future.

Reply:
"amd64 ~arm ~hppa ppc ~ppc64 x86" - I copied it from app-misc/worker, yes.
Ah! Noticed now that in app-misc/worker I made all testing (added "~"), but forgot to do the same for keys-worker.